modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -156,7 +156,7 @@ private Boolean getGeneratorPropertyDefaultSwitch(final String key, final Boolea
         return defaultValue;
     }
 
-    private void configureGeneratorProperties() {
+    protected void configureGeneratorProperties() {
```

Comment:
This should not be protected. If you're doing this for testing, please remove the modifier and treat this as package-private. By adding `protected`, you're telling users that we expect them to customize this and modify the method. That would be a maintenance nightmare for us if we don't even know how the tool's properties are applied.

Reply:
Done